projects/igniteui-angular/src/lib/core/styles/components/grid/_excel-filtering-theme.scss

```diff
@@ -399,12 +404,39 @@
 
         &:hover,
         &:focus {
-            background: color($color: 'gray', $variant: 100);
             color: var-get($theme, 'excel-filtering-actions-hover-foreground');
+
+            @if $variant == 'indigo' {
+                @if $theme-variant == 'light' {
+                    background: color($color: 'gray', $variant: 200);
+                } @else {
+                    background: contrast-color($color: 'gray', $variant: 50, $opacity: .1);
+                }
```

Comment:
Again, this ideally comes from the schema, not something that is hardcoded here.